tests/estimators/speech_recognition/test_pytorch_deep_speech.py

```diff
@@ -356,6 +358,10 @@ def _test_all(self, request, setup_class):
 
         # Now test fit function
         if request.param is True:
+            # Create the optimizer
```

Comment:
In general there is a few simple but important changes that need to be performed on this test to conform to the rest of ART standard tests. 1/ tests should not be placed within a class `class TestPyTorchDeepSpeech` 2/ the `master_seed(seed=1234)` needs to be remove as it is set in 1 location only for the entire set of tests 3/ all of the large np.asarrays should be stored and retrieved using the `store_expected_values` and `expected_values` ART fixtures to increase visibility. I can help out on how to use them if necessary, It's really easy

Reply:
Solved.